test/relations.test.js

```diff
@@ -34,9 +34,16 @@ describe('relations', function() {
     db = getSchema();
   });
 
+  after(function(done) {
+     // disconnect from this db to avoid too many connection error
+     // due to multiple instance of connection pool
+    db.disconnect();
+    done();
+  });
+
   describe('hasMany', function() {
     before(function(done) {
-      db = getSchema();
+      // db = getSchema();
```

Comment:
The `getSchema()` implementation is from the connector. Most of our connectors test return a singleton to avoid overwhelming number of connections to the database. As a result, we don't call db.disconnect() unless you're sure the db is an isolated instance.